src/main.rs

```diff
@@ -158,7 +160,67 @@ pub fn ask_to_run(mut cmd: Command, ask: bool, text: &str) {
     }
 }
 
-fn build_sysroot(auto: bool, target: &str, rustc_version: &VersionMeta) -> PathBuf {
+/// Returns the sanitizer to use.
+/// Ok(Some(str)) => Use sanitizer "string"
+/// Ok(None) => Use no sanitizer
+///
+/// # Errors
+/// Err(err) => There was an error when trying to get the list of supported sanitizers
+pub fn sanitizer(target: &str) -> anyhow::Result<Option<&'static str>> {
```

Comment:
Please import `anyhow::Result`

Reply:
I usually prefer leaving this explicit, to differentiate from non-typedef'd `Result`. But I've imported it as you requested

Reply:
Yeah this is definitely a matter of personal preference. When there is only one `Result` in use in a project, I prefer to import it.